kornia/color/gray.py

```diff
@@ -32,52 +36,54 @@ def grayscale_to_rgb(image: Tensor) -> Tensor:
     return concatenate([image, image, image], -3)
 
 
-def rgb_to_grayscale(image: Tensor, rgb_weights: Tensor | None = None) -> Tensor:
+def grayscale_from_rgb(
```

Comment:
Shouldn't we remove the typing from image? np.ndarray is not a tensor

Reply:
i think we should map KerasTensor which is already an alias of the subset of supported tensor/ndarrays from the different frameworks. We can discover during the migration what's best and what give less headaches during development